core/src/vortexor_receiver_adapter.rs

```diff
@@ -0,0 +1,131 @@
+//! Vortexor receiver adapter which wraps the VerifiedPacketReceiver
+//! to receive packet batches from the remote and sends the packets to the
+//! banking stage.
+
+use {
+    crate::banking_trace::TracedSender,
+    agave_banking_stage_ingress_types::BankingPacketBatch,
+    crossbeam_channel::{unbounded, Receiver, RecvTimeoutError, Sender},
+    solana_perf::packet::PacketBatch,
+    solana_vortexor_receiver::receiver::VerifiedPacketReceiver,
+    std::{
+        net::UdpSocket,
+        sync::{atomic::AtomicBool, Arc},
+        thread::{self, Builder, JoinHandle},
+        time::{Duration, Instant},
+    },
+};
+
+// Macro to send the packet batch to the sender
+macro_rules! send {
```

Comment:
odd to use a macro for this imo - especially if we get rid of the logging.

Reply:
changed to a function